### PR TITLE
Tree labels

### PR DESCRIPTION
**Pull request by mirunaish**

### CODE REVIEW

File: packages/schema/src/tree.ts [hunk 1]
```diff
@@ -71,6 +71,11 @@ export interface Tree {
    */
   speciesCode?: string;
 
+  /**
+   * Labels.
+   */
```
Comment (julian-george) [inline packages/schema/src/tree.ts:76]: Maybe a bit more of a description here

**Review by julian-george — APPROVED** [review]

File: packages/server/src/db/seeders/20220125031200-main.js [hunk 2]
```diff
@@ -269,6 +306,7 @@ module.exports = {
       await queryInterface.bulkDelete("trees", null, { transaction });
       await queryInterface.bulkDelete("tree_species", null, { transaction });
       await queryInterface.bulkDelete("tree_statuses", null, { transaction });
+      await queryInterface.bulkDelete("tree_labels", null, { transaction });
       await queryInterface.bulkDelete("users", null, { transaction });
       await queryInterface.bulkDelete("plots", null, { transaction });
       await queryInterface.bulkDelete("trips", null, { transaction });
```
Comment (ziruihao) [inline packages/server/src/db/seeders/20220125031200-main.js:312]:
```suggestion
      await queryInterface.bulkDelete("tree_tree_label", null, { transaction });
      await queryInterface.bulkDelete("trees", null, { transaction });
      await queryInterface.bulkDelete("tree_species", null, { transaction });
      await queryInterface.bulkDelete("tree_statuses", null, { transaction });
      await queryInterface.bulkDelete("tree_labels", null, { transaction });
      await queryInterface.bulkDelete("users", null, { transaction });
      await queryInterface.bulkDelete("plots", null, { transaction });
      await queryInterface.bulkDelete("trips", null, { transaction });
```

**Review by ziruihao — APPROVED** [review]
Amazing work for your first PR! Thanks for diving straight into it and being super independent. Code is very clean and I appreciate you followed the current conventions in the repository!

File: packages/server/src/db/seeders/20220125031200-main.js (hunk 2 above)
Comment (ziruihao) [reply packages/server/src/db/seeders/20220125031200-main.js:312]: Just have to also clear the `tree_tree_label` table on seeding undo!

**Pull request merged**